docs/reference/commandline/ps.md

```diff
@@ -60,5 +61,38 @@ The currently supported filters are:
 
 This shows all the containers that have exited with status of '0'
 
-
-
+## Formatting
+
+The formatting flag (`-F` or `--format`) will pretty-print container output using a Go template.
+
+Valid placeholders for the Go template include:
+* .ID - Container ID
```

Comment:
I think the placeholders should be in ticks.

I'm thinking that a table would be a nicer way of presenting this than a bullet-list, e.g.

Placeholder | Description
---- | ----
`.ID` | Container ID
`.Image` | Image ID